src/app/armory/wishlist-collapser.ts

```diff
@@ -1,11 +1,14 @@
 import { D2ManifestDefinitions } from 'app/destiny2/d2-definitions';
 import { DimItem } from 'app/inventory/item-types';
+import { HashLookup } from 'app/utils/util-types';
 import { WishListRoll } from 'app/wishlists/types';
 import { DestinyInventoryItemDefinition, TierType } from 'bungie-api-ts/destiny2';
 import { ItemCategoryHashes } from 'data/d2/generated-enums';
-import perkToEnhanced from 'data/d2/trait-to-enhanced-trait.json';
+import perkToEnhancedFile from 'data/d2/trait-to-enhanced-trait.json';
 import _ from 'lodash';
 
+const perkToEnhanced: HashLookup<number> = perkToEnhancedFile;
```

Comment:
suggest instead

trait-to-enhanced-trait.d.ts
```ts
declare module 'data/d2/trait-to-enhanced-trait.json' {
  const x: NodeJS.Dict<number>;
  export default x;
}
```

which we should really probably be publishing with all .json files in d2ai, cc @delphiactual